types/lodash/lodash-tests.ts

```diff
@@ -4173,6 +4167,19 @@ fp.now(); // $ExpectType number
     } else {
         obj2; // $ExpectType { value: boolean; }
     }
+    type Errors = Record<string, string>;
+    const errors: Errors = {};
+
+    if (_.isEmpty(errors)) {
+        errors.test = "123";
+    }
+    const bag = {};
```

Comment:
I would rather remove this test - it doesn't seem like a valid use case. (Or maybe replace this test with the use case in my other comment)

Reply:
This is pulled from https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/62379 - yeah, replacing it with the test case you proposed is my preference too.